src/clj/rems/css/styles.clj

```diff
@@ -96,8 +102,8 @@
                            :background-size :contain
                            :background-repeat :no-repeat
                            :background-position [[:center :center]]}]
-                         [:.logo
-                          {:height (u/px 150)}]))
+                         [:.logo {:height logo-height}]
+                         [:.logo-menu {:display "none"}]))
```

Comment:
I think we need to adjust these breakpoints more. It doesn't quite work with this. I would rebase on top of the master (with the removed Profile-link) and then fix the breakpoint but we can also merge this change and do another PR with these fixes. Definitely should adjust the breakpoint with the latest change to the navbar.

![breakpoint_failure](https://user-images.githubusercontent.com/823661/106110752-caf27800-6153-11eb-9b69-a81a2c8dadd9.png)

Also the menu here should be on the left so we should hide the logo earlier.
![slightly_bigger_logged_in](https://user-images.githubusercontent.com/823661/106110825-e3fb2900-6153-11eb-8719-97d448114fd3.png)

Reply:
I personally do not rely on Chrome simulator much, been known to display layout with 'quirks'
I would go and test it properly with Xcode and Android studio for the different PR.
@joel and @Macroz do we have any usage stats? Like desktop vs. mobile usage I can fall back on to make a decision how to test?

Reply:
No stats :/

Reply:
ok, so no analytics I take it?

Reply:
Yeah, REMS doesn't include any analytics, and if any users have included their own, we haven't seen the data.

Reply:
Simulator or not a person can easily have a window of inopportune width, you can try the same with resizing your browser window. The responsiveness of the page should not be broken.

Reply:
I think I can create an issue from this discussion